airbyte-cdk/python/pyproject.toml

```diff
@@ -66,7 +66,9 @@ pytz = "2024.1"
 [tool.poetry.group.dev.dependencies]
 freezegun = "*"
 mypy = "*"
-pandas = "2.0.3"
+asyncio = "*"
```

Comment:
why is this new dependency needed?

Reply:
It is used in https://github.com/airbytehq/airbyte/blob/2eaccd00433a32bfd8f904fe69621946cd066136/airbyte-cdk/python/airbyte_cdk/sources/file_based/stream/default_file_based_stream.py#L5